modules/tinymce/src/core/main/ts/fmt/RemoveFormat.ts

```diff
@@ -25,6 +25,31 @@ import * as CaretFormat from './CaretFormat';
 import * as ExpandRange from './ExpandRange';
 import * as FormatUtils from './FormatUtils';
 import * as MatchFormat from './MatchFormat';
+import { mergeSiblings } from './MergeUtils';
+
+interface RemoveFormatAdt {
```

Comment:
I thought I'd heard somewhere that we were trying to move away from the Adt module? Was that incorrect?

Reply:
We're moving away from using it where it's not needed, that's about it. `Struct` was the one we wanted to remove.

Reply:
I should add the reason I'm using it here is this can't be solved with a simple enum, or something like that. We need to know what to do, as well as know additional data about what to do (eg what to rename as). I could have got away with just using `Rename` as an enum, but then it's not clear what it should be renamed as.

I guess I could have used something like `RenameToSpan` but that just felt messy.